tests/components/aws/test_init.py

```diff
@@ -69,22 +69,20 @@ async def test_empty_credential(hass):
         )
         await hass.async_block_till_done()
 
-    sessions = hass.data[aws.DATA_SESSIONS]
-    assert sessions is not None
-    assert len(sessions) == 1
-    session = sessions.get("default")
-    assert isinstance(session, MockAioSession)
-
     assert hass.services.has_service("notify", "new_lambda_test") is True
     await hass.services.async_call(
```

Comment:
We should probably make the service call inside the patch context manager.

Reply:
oh lol

Reply:
It would already be registered in that case?